src/main/javascript/crypto/src/libs/AttestedObject.ts

```diff
@@ -21,16 +15,20 @@ export class AttestedObject {
     private unsignedEncoding: string;
     private derEncodedProof: string;
     private signature: string;
-    constructor(private attestableObject: Cheque, private att: SignedAttestation,
-                // private keys: KeyPair,
-                private attestationSecret: bigint ,private chequeSecret: bigint) {
+    constructor(
+        private attestableObject: AttestableObject,
+        private att: SignedAttestation,
+        // private keys: KeyPair,
+        private attestationSecret: bigint ,
+        private chequeSecret: bigint
+    ) {
         this.crypto = new AttestationCrypto();
         // this.userPublicKey = userKeys.getPublic();
         this.pok = this.makeProof(attestationSecret, chequeSecret, this.crypto);
         this.derEncodedProof = this.pok.getDerEncoding();
 
         let vec =
-            uint8tohex(this.attestableObject.getDerEncoding()) +
+            this.attestableObject.getDerEncoding() +
             uint8tohex(this.att.getDerEncoding())+
```

Comment:
Why is this DER encoding encoded to hex before being stored as part of the DER encoding of the whole object?

Reply:
getDerEncoding() returns string now